students/luftsmeerflier/lesson04/mailroom_3.py

```diff
@@ -0,0 +1,164 @@
+#/usr/bin/env python3
+import sys
+
+donors = [
+	#The actual donations themselves are a list of objects: Donations.date and Donations.amount
+	{
+		"first_name": "John",
+		"last_name": "Quigley",
+		"address": "555 Camelot RD N London, Ontario",
+		"donations": 
+			[
+				("2019-02-04", 5000.00),
+				("2019-02-04", 4250.00),
+				("2019-02-04", 3000.00)
+		
+			]
+	},
+	{
+		"first_name": "Sara",
+		"last_name": "Smith",
+		"address": "256 Camden RD N London, UK",
+		"donations": [
+			("2020-04-02", 2000.00),
+			("2019-02-04", 42000.00),
+			("2019-02-04", 32000.00)
+		]
+	},
+	{
+		"first_name": "Jacob",
+		"last_name": "van der Schmidt",
+		"address": "2001 Space Oddysey RD SW Los Angeles, CA",
+		"donations":[
+			("2012-04-20", 20.00),
+			("2019-02-04", 2000.00),
+			("2019-02-04", 3400.00)
+		]
+	},
+	{
+		"first_name": "Jason",
+		"last_name": "Quigley",
+		"address": "12 Camden RD N London, UK",
+		"donations": [
+			("2012-02-04", 12.00),
+			("2013-02-04", 240.00),
+			("2014-02-05",120.00)
+		]
+	},
+	{
+		"first_name": "Ogden",
+		"last_name": "Nash",
+		"address": "256 Camden RD N London, UK",
+		"donations": [
+			("2019-02-04", 5000000.00),
+			("2019-02-04", 500000.00),
+			("2019-02-04", 50000.00)
+		]
+	}
+]
+
+def send_thank_you(first_name='', last_name=''):
+	list = input("Type `list` for a list of donors or press the carriage key to skip\n")
+	if list == "list":
+		print('\n')
+		for donor in donors:
+			print(donor['first_name'], donor['last_name'])
+		print('\n')
+		return
+	else:
+		pass
```

Comment:
please correct your indentation to be 4 spaces (lint + style is part of the grade rubric)